packages/ui/src/theme/overwrites/components/Checkbox.ts

```diff
@@ -0,0 +1,41 @@
+function baseStyleControl() {
```

Comment:
🤔 Now there is probably no need to use a function here and for `baseStyle`. You can simply create objects.